src/status_im/contexts/wallet/add_address_to_watch/view.cljs

```diff
@@ -115,24 +115,25 @@
       [rn/view
        {:style {:flex 1}}
        [floating-button-page/view
-        {:header [quo/page-nav
-                  {:type      :no-title
-                   :icon-name :i/close
-                   :on-press  (fn []
-                                (rf/dispatch [:wallet/clean-scanned-address])
-                                (rf/dispatch [:wallet/clear-address-activity-check])
-                                (rf/dispatch [:navigate-back]))}]
-         :footer
-         [quo/button
-          {:customization-color customization-color
-           :disabled?           (or (string/blank? @input-value) (some? (validate @input-value)))
-           :on-press            (fn []
-                                  (rf/dispatch [:navigate-to
-                                                :confirm-address-to-watch
-                                                {:address @input-value}])
-                                  (clear-input))
-           :container-style     {:z-index 2}}
-          (i18n/label :t/continue)]}
+        {:keyboard-suggestion-bar? true
```

Comment:
we should add this to transaction-progress page too 👍

Reply:
With the new approach you suggested, this is now needed. Updated the code to set `footer-container-padding` to `0` where needed.